tm2/pkg/iavl/tree_fuzz_test.go

```diff
@@ -107,19 +121,179 @@ func genRandomProgram(size int) *program {
 func TestMutableTreeFuzz(t *testing.T) {
 	t.Parallel()
 
+	runBasicMutableTreeFuzzing(t)
+}
+
+var pathForMutableTreeProgramSeeds = filepath.Join("testdata", "corpra", "mutable_tree_programs")
+
+func runBasicMutableTreeFuzzing(tb testing.TB) {
+	tb.Helper()
+	runThenGenerateMutableTreeFuzzSeeds(tb, false)
+}
```

Comment:
Why this function?

Reply:
It was there to make more it much more sensible to give context to the readers, but yeah let me remove it. Thanks!